### PR TITLE
Add files

### PR DESCRIPTION
**Pull request by vaijab**
Why Fedora? Because CentOS does not have redis package.

### CODE REVIEW

**Comment by PurpleBooth** [comment]
Can't you just compile it?

**Comment by vaijab** [comment]
> Can't you just compile it?

I could. But why bother? Why include gcc and other build tools in the image unnecessarily.

**Comment by PurpleBooth** [comment]
Because it means you can use the proper base image?

You can add the build tools you need, then uninstall them in the same layer, it's not hard.

**Comment by allandegnan** [comment]
EPEL?

**Comment by vaijab** [comment]
> EPEL?

epel provides an old version of redis which is 2.8.x. Current stable version is 3.0.x.

**Comment by allandegnan** [comment]
Stable is a relative term. From the software perspective, stable is the latest version.  From an OS perspective, it's the version you get with an stable distro that doesn't update the major version as that's likely to introduce breaking changes.

Personally, I don't give a damn about what version of Redis I have as long as it does what Redis is supposed to do.  Does 3.0 come with features you need that aren't in 2.8?  Great, you have a fantastic argument and i agree with you.  If it doesn't then go with the stable OS version because we don't need to be patching it in 6-12 months when the current version of fedora is no longer supported.

So basically, if you want to push fedora here you need to validate it slightly better than "this is the stable version".

**Comment by vaijab** [comment]
> Does 3.0 come with features you need that aren't in 2.8?

@allandegnan yes, let me clarify this :-). When I tested redis sentinel in v2.8 I found lots of issues and it was not as stable as one in v3.0.

@PurpleBooth compiling from source is something I'd really like to avoid. I am OK with downloading a tarball which contains binaries though, but redis does not provide that.
